tests/common/util.rs

```diff
@@ -3611,7 +3611,7 @@ mod tests {
         assert!(command.tmpd.is_some());
     }
 
-    #[cfg(all(unix, not(target_os = "macos")))]
+    #[cfg(all(unix, not(any(target_os = "macos", target_os = "openbsd"))))]
     #[test]
     fn test_compare_xattrs() {
```

Comment:
That would mean that `compare_xattrs` remains untested but (possibly) used on OpenBSD. That sounds like a bad idea. Can you simply `cfg` out the function `compare_xattrs`, too? Does that work?

Reply:
Right and thanks for pointing to it. Added the `cfg` condition to the `compare_xattrs` function too and updated the commit.